app/views/items/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with model: @item do |form| %>
+<%= form.label :name, "Item name : " %>
+<%= form.text_field :name %>
+<br>
+<br>
+<%= form.label :quantity, "quantit : " %>
```

Comment:
spelling

Reply:
quantit -> Quantity and also resolve to spaces